### ISSUE TITLE
[Feature]add support for sharing images to other apps

### ISSUE THREAD

**Issue by pokegamer5547** [leonlatsch/Photok#103]
**Is your feature request related to a problem? Please describe.**
Sharing image in app is not possible and it can be a deal breaker when I want to share it to a specific app only.
**Describe the solution you'd like**
Having a share option in app to other image to other apps like chatting apps. 
**Additional context**
Maybe using contact provider of andoid night enable users to securely share images to other apps(as long as the temporary file path is not accessible). I'm not sure how exactly this works but there's an app called EDS lite which does similar thing and overwrite temporary data once app is closed.

**Comment by leonlatsch** [comment]
Good Idea, I will have a look into that, but for now this is backlog.

Btw. I don't have any time for this projet atm, because I'm finishing my degree right now.
I hope I will have time for Photok again later this year.

**Comment by pokegamer5547** [comment]
@leonlatsch oh I see, I can understand, it's alright. maintaining  life while building softwares isn't an easy thing. all the best for your degree :)

**Comment by hellyeahmfs** [comment]
Yeah. This is a deal breaker for me. Hope this is comes very soon. 🤩
